scripts/create-component/index.js

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env node
```

Comment:
are there cross-OS problems with this.  Not familiar enough to know.  would the script reading `node thisscript.js` be more sure-fire compatible?

Reply:
i don't quite know the difference. you'll have to teach me

Reply:
just that the npm-script for create:component could have the value: "node scripts/create-component/index.js"

but again, not 100% sure that this current version is not windows compat.

Reply:
oh gotcha - yeah i can change that